src/main/java/com/autotune/queue/ExpMgrQueue.java

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
```

Comment:
I think the EM related Queue should ideally be inside the EM directory and should not be part of this PR. We just need a generic interface here that is not specific to any module and can be extended / implemented by the required modules.

Reply:
AutotuneQueueFactory using both the instances so I have to put it as part of this PR if you want it in separate package then I have create and move this class.
Please let me know your decision?